app/views/backend/fixed_assets/show.html.haml

```diff
@@ -40,13 +40,24 @@
               = link_to(purchase_item.journal_entry.number, {controller: :journal_entries, action: :show, id: purchase_item.journal_entry.id})
 
   = infos do
-    - if resource.sold_on
+    - if resource.sold? && resource.sold_on
```

Comment:
Sell date is mandatory if the fixed asset is sold. `if resource.sold?` is enough here.